Gemfile

```diff
@@ -47,6 +47,7 @@ gem 'raise-if-root'
 gem 'readthis'
 gem 'recaptcha', require: 'recaptcha/rails'
 gem 'redis-session-store', '>= 0.11.3'
+gem 'retries', '0.0.5'
```

Comment:
no longer needed, now that the mock job lives in the gem right?

Reply:
I think it's needed either way, since we use retries in both the mock lambda and the real one and we don't install any of the lambda dependencies.

Reply:
Oh whoops, I should have added inside the `.gemspec` in the top-level --- I'll make a point to do that to clean up